### PR TITLE
Add D10D - Piunora

### PR DESCRIPTION
**Pull request by timonsku**
Piunora is an open source carrier for the Raspberry Pi Compute Module 4 which offer USB device functionality.

### CODE REVIEW

File: 1209/D10D/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: Piunora
+owner: Diodes Delight
```
Comment (tannewt) [inline 1209/D10D/index.md:4]:
```suggestion
owner: Diodes-Delight
```

**Review by tannewt — CHANGES_REQUESTED** [review]
Please mention in the hardware readme that it'll run: https://github.com/tannewt/circuitpython/tree/piunora AKA, that's what the PID is for and it is open source.

File: 1209/D10D/index.md (hunk 1 above)
Comment (timonsku) [reply 1209/D10D/index.md:4]: If due formatting restrictions thats fine! otherwise its not written with a -

**Comment by timonsku** [comment]
Ah I missed that the PID readme includes HW description. Added now.

File: 1209/D10D/index.md (hunk 1 above)
Comment (tannewt) [reply 1209/D10D/index.md:4]: It has to match the filename in `org/`

Comment (tannewt) [reply 1209/D10D/index.md:4]: (That's why the CI is failing.)

**Review by tannewt — APPROVED** [review]

**Pull request merged**